web/main/colors.css

```diff
@@ -45,6 +50,7 @@
       0 -1px oklch(0% 0 0 / 20%);
     --field-color: oklch(100% 0 0);
     --field-shadow: inset 0 1px 2px oklch(from var(--shadow-color) l c h / 8%);
+    --link-color: oklch(49.77% 0.224 271.1);
```

Comment:
It is possible to take color from Harmony?
https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.figma.com/community/file/1287828769207775946/harmony-accessible-ui-color-palette&ved=2ahUKEwjppZyO8qOFAxXQQUEAHewQDu0QFnoECBQQAQ&usg=AOvVaw0BjztSBE5b3U7HmRkyQdM3

Reply:
Omg how I missed this tool, awesome, yes let's take color from Harmony. Should we use these colors in most cases? If so, maybe we should add some notes to the 'Visual Language' section in the documentation.